### PR TITLE
Restart synchronizing TTL status when the connection is lost

### LINKED ISSUE #263: Support retrying connection in case connection for TTL status is lost
> Let's support retrying connection in case the connection is lost.
You can separate it as another issue if you want.

_Originally posted by @kangz12345 in https://github.com/snu-quiqcl/iquip/pull/247#discussion_r1537628882_

### PR DESCRIPTION
**Pull request by BECATRUE**
This closes #263.

Now, ARTIQ is in use, so I cannot test this PR..

### CODE REVIEW

File: iquip/apps/monitor.py [hunk 1]
```diff
@@ -218,10 +223,16 @@ def run(self):
...
+                    try:
+                        response = websocket.recv(5)
+                    except TimeoutError:
+                        if websocket.ping().wait(5):
+                            continue
+                        break  # connection is lost
                     status = json.loads(response)
                     self.fetched.emit(status)
-        except WebSocketException:
+        except:  # pylint: disable=bare-except
```
Comment (kangz12345) [inline iquip/apps/monitor.py]: Please do not disable `bare-except`.

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: iquip/apps/monitor.py (hunk 1 above)
Comment (BECATRUE) [reply iquip/apps/monitor.py]: Okay. I will apply your review in #267.

File: iquip/apps/monitor.py [hunk 2]
```diff
@@ -218,10 +223,16 @@ def run(self):
...
+                    try:
+                        response = websocket.recv(5)
+                    except TimeoutError:
+                        if websocket.ping().wait(5):
+                            continue
+                        break  # connection is lost
                     status = json.loads(response)
                     self.fetched.emit(status)
-        except WebSocketException:
+        except Exception:
```
Comment (kangz12345) [inline iquip/apps/monitor.py]: Now you have to disable `broad-exception-caught` 😄

**Review by kangz12345 — CHANGES_REQUESTED** [review]

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**